libexec/bats-core/bats-exec-test

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env bash
 set -eET
 
+# Variables used in other scripts.
```

Comment:
explicitly stating that these variables are coming from the outside of this scrtipt